Example/Controllers/SwiftUI/Features/Terms/MiniAppWithTermsView.swift

```diff
@@ -35,12 +36,21 @@ struct MiniAppWithTermsView: View {
             }
         }
         .alert(isPresented: $showMessageAlert) {
-            Alert(
-                title: Text("Info"),
-                message: Text("Message sent!"),
-                dismissButton: Alert.Button.cancel(Text("Ok"), action: {
-                    viewModel.showMessageAlert.send(false)
-            }))
+            if let message = viewModel.showJsonString {
+                return Alert(
+                    title: Text("Universal Bridge"),
```

Comment:
Is this the message received from MiniApp ?

Reply:
Yes this is the message received from miniapp for interface1 implementation.